### ISSUE TITLE
Shell support

### ISSUE THREAD

**Issue by matz3ibm** [scgray/jsqsh#61]
Hi Scott,

probably not found the right workaround or command yet, but the Operations team and I were looking on how to switch to shell mode from jsqsh, eg. for doing some researching in hdfs, run some scripts or ETL jobs. In the \help I found:
"2> go | grep test | more" 
.... but would like to have something like 
>\sh to invoke a "configurable shell"
to execute bash, ksh, sh, probably with cmd line as parameter- similar to vi ":sh" command.

Using with alias command with template mechanism would be very practible: 
\alias listdnodes @@cluster@@='\sh ./opt/IBM/tools/listnodes.pl -s=dnodes -c @@cluster@@'

(I think there is a little error on the alias wiki page, "pre-argumented commands": https://github.com/scgray/jsqsh/wiki/alias  in example: 1> \alias '\prod'='\connect -U sa -S prod-db -d mssql-jtds' ==> then it's called with "1>\prod", or? - but would tat not be improvable like 1> \alias '\env:@@env@@'='\connect -U sa -S @@env@@ -d mssql-jtds' ===> called "1>\env:prod-db"

regards from Frankfurt,
Matthias

**Comment by scgray** [comment]
Yes, you are right that the documentation for `\alias` has a bug in it.  It should have just been `\prod` not `\prod-db` in the example.  I'll fix that.

A command to execute an arbitrary binary should be relatively easy.  I'll see if I can take a look at it over the weekend.  

I hope all is well!
